WalkingChallenge/DropDownPicker.swift

```diff
@@ -0,0 +1,195 @@
+
+import UIKit
+
+class DropDownPicker: UIControl {
+  fileprivate var picker: UIPickerView!
```

Comment:
I would make these `let`s instead and initialize them from the start and just configure in the init